src/app/view/components/_common/dot-site-selector/dot-site-selector.component.ts

```diff
@@ -169,11 +173,7 @@ export class DotSiteSelectorComponent implements OnInit, OnChanges, OnDestroy {
         this.paginationService.filter = filter;
         this.paginationService.getWithOffset(offset).subscribe((items) => {
             this.sitesCurrentPage = [...items];
-            this.totalRecords = this.totalRecords || this.paginationService.totalRecords;
```

Comment:
this is not needed ?

Reply:
No, just used the `paginationService.totalRecords` directly.